src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -123,7 +129,27 @@ const SubscriberHistory = ({ location, dependencies: { dopplerApiClient } }) =>
                     <>
                       {state.sentCampaigns.map((campaign, index) => (
                         <tr key={index}>
-                          <td>{campaign.campaignName}</td>
+                          <td>
+                            {campaign.imgPreview ? (
+                              <div className="dp-tooltip-container">
+                                <a
+                                  href={`https://reports2.fromdoppler.com/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
+                                >
+                                  {campaign.campaignName}
+                                  <div className="dp-tooltip-block">
+                                    <img src={campaign.imgPreview.href} alt="" />
```

Comment:
```suggestion
                                    <img src={campaign.imgPreview.href} alt="Campaign Preview" />
```
Or better add this into a resourced text.

Reply:
fixed